plugins/catalog-backend/src/service/DefaultRefreshService.ts

```diff
@@ -45,4 +49,9 @@ export class DefaultRefreshService implements RefreshService {
       });
     });
   }
+  async refreshByRefreshKey(options: RefreshByRefreshKeysOptions) {
```

Comment:
Worth renaming this method and make it accept multiple keys.

```suggestion
  async refreshByRefreshKeys(options: RefreshByRefreshKeysOptions) {
```